src/FileReader.java

```diff
@@ -0,0 +1,31 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+
+class FileReader {
+
+    /* Checking if the file is either txt or json  */
+
+    public static boolean isFileExists(String filepath) {
+        if (filepath.endsWith("txt") || filepath.endsWith("json")) {
+            readingFileContent(filepath);
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public static void readingFileContent(String filePath) {
+        try {
+            List<String> allLines = Files.readAllLines(Paths.get(FileSearchApplication.filepath));
```

Comment:
When you passed filePath as argument to the readingFileContent, instead of using it as parameter to Paths.get method, why are you getting filePath from "FileSearchApplication.filepath" ? If you want to use "FileSearchApplication.filepath", then I see filePath argument to the method readingFileContent makes no sense .